persistence/album_repository.go

```diff
@@ -156,6 +156,7 @@ func (r *albumRepository) refresh(ids ...string) error {
 		Years         string
 		DiscSubtitles string
 		Path          string
+		Size          int
```

Comment:
This line is not required and can be removed. This `refreshAlbum` struct is for extra columns that are needed for post-processing the query results, if the value for a field in the result set is one-to-one to the expected album value, you don't need to do anything else.